packages/template-cli/README.md

```diff
@@ -26,67 +26,111 @@ Creates an amidostacks template in your local directory much like express-genera
 
-6. To test the deploy folder has been correctly provisioned prior to checking you need to at this point in time copy over a sample backend-config and terraform vars. Currently vars.tf and provider configuration is not automatically updated. 
+7. To get a local deployment open the following files and uncomment these lines
+
+`${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/deploy/terraform/azure/provider.tf`
+```terraform
+   # resource_group_name  = "amido-stacks-rg-uks"
+   # storage_account_name = "amidostackstfstategbl"
+   # container_name       = "tfstate"
+   # # key                  = "spa-cycle-2"
+```
+
+`${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/deploy/terraform/azure/vars.tf`
+```terraform
+
+# locals {
+#   # // common //
+#   resource_prefix           = "${var.name_company}-${var.name_project}-${var.name_component}"
+#   resource_suffix           = "${var.location_name_map[var.resource_location]}-${var.name_environment}"
+
+#   # // resource group //
+#   resource_group_name       = "${local.resource_prefix}-rg-${local.resource_suffix}"
+
+#   # // storage account - static website hosting //
+#   storage_account_name      = "${var.name_company}${var.name_project}${var.name_component}sa${var.location_name_map[var.resource_location]}${var.name_environment}"
+# }
+```
+
+8. To test the deploy folder has been correctly provisioned prior to checking you need to at this point in time copy over a sample backend-config and terraform vars. Currently vars.tf and provider configuration is not automatically updated. 
 Future iterations will include this.
 
 The safest way to run and maintain this going forward is to rely on environment variables for credentials as that is the way the pipeline will trigger the executions of terraform. 
 
 Sample export script with correct environment vars:
 
 ```bash
-export ARM_CLIENT_ID= \
+$ export ARM_CLIENT_ID= \
 ARM_CLIENT_SECRET= \
 ARM_SUBSCRIPTION_ID= \
 ARM_TENANT_ID=
-```
 
+$ echo "
+vnet_id                 = \"amido-stacks-vnet-uks-dev\"
+rg_name                 = \"amido-stacks-rg-uks-dev\"
+resource_group_location = \"uksouth\"
+name_company            = \"amido\"
+name_project            = \"stacks\"
+name_component          = \"spa\"
+name_environment        = \"dev\"
+" > ${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/deploy/terraform/azure/backend.local.tfvars
 ```
-cd ${YOUR_GIT_STACKS_WEB_APP_PATH}/deploy/azure/terraform
-terraform init -backend-config=./backend.local.tfvars
-terraform plan
+
+
+```bash
+$ cd ${YOUR_GIT_STACKS_WEB_APP_PATH}/deploy/terraform/azure
+$ terraform init -backend-config=./backend.local.tfvars
+$ terraform plan
 ```
 
 NB: as the module currently points to a private github repo you will need to supply credentials on the command line as part of the plan/apply
 
 ## Testing
 
 ```bash
-npm run test
+$ cd ${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/packages/template-cli
+$ npm run test
```

Comment:
While this is true, lerna will automatically run all the tests for this package from the root.

Reply:
do you want me to remove the test entirely?

Reply:
removed